tools/boilerplate/plop-templates/package/test/jest.setup.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { ThemeProvider } from 'styled-components'
```

Comment:
just use `ThemeProvider` from `pcln-design-system`, then you don't even need `createTheme()`

Reply:
No longer needed since we removed the global stuff